docs/en/integrations/dbt/dbt-intro.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_label: Introduction 
+sidebar_position: 1
+description: Users can transform and model their data in ClickHouse using dbts
+---
+
+# ClickHouse and dbt
+
+dbt (data build tool) enables analytics engineers to transform data in their warehouses by simply writing select statements. dbt handles materializing these select statements into objects in the database in the form of tables and views - performing the T of ELT. Users can create a model defined by a SELECT statement.
```

Comment:
ELT is a newish acronym in the industry. We can detail Extract Load and Transform and maybe link a wikipedia page or something